src/sparseml/experimental/sparsegpt/utils.py

```diff
@@ -240,9 +240,9 @@ def get_wikitext2(nsamples, seed, seqlen, model):
     traindata = load_dataset("wikitext", "wikitext-2-raw-v1", split="train")
     testdata = load_dataset("wikitext", "wikitext-2-raw-v1", split="test")
 
-    from transformers import AutoTokenizer
+    from sparseml.transformers import SparseAutoTokenizer
```

Comment:
We shouldn't be messing with any of the code in sparseml/experimental, its research code